08-components-lab/src/02-numberguessing.jsx

```diff
@@ -1,17 +1,46 @@
-/**
- * Lab 02:
- *
- * Implement a number guessing game component.
- *
- * The component should provide an input field for the user to guess a
- * number picked randomly by the computer (from 1 to 1,000).
- * When typing a number, the component should print "too low" or "too high" 
- * as a hint to the user.
- *
- * To make the game interesting, computer should sometimes cheat and print wrong messages.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+import React from "react";
+
+export default class NumberGuessing extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            userGuess: 0,
+            randomNumber: Math.floor((Math.random() * 1000) + 1),
+            validateMsg: "",
+        };
+
+        console.log(this.state.randomNumber);
+    }
+
+    newNumberGuess(event) {
+        var userGuess = Number(event.target.value);
```

Comment:
You can use ES6 function arguments destructuring here to get `event.target.value` inside function arguments